requirements/connectors/duckdb.in

```diff
@@ -0,0 +1,3 @@
+-r ../common/base.in
+
+duckdb==1.1.0
```

Comment:
This is great for the requirements. Could you please add a line in this file:
https://github.com/Unstructured-IO/unstructured-ingest/blob/b3add2abf347b5e21121bf01626f8b4e9e3ec9e1/setup.py#L113

It is what gives us the ability for the user to run `pip install "unstructured-ingest[duckdb]"

Reply:
@potter-potter would this : https://github.com/krish-adi/unstructured-ingest/blob/625797062d3b151d896d1a4a4cea39dbb06b889e/setup.py#L97  work?

Reply:
I think that is it. I'll let you know if theres more. (Never remember this part. I always just search for `pinecone` and see where we added them.